docs/configuration.md

```diff
@@ -0,0 +1,72 @@
+# Elemental Grid Configuration
+
+## Default settings
+Use the following yaml configuration to override defaults;
+```yaml
+TheWebmen\ElementalGrid\ElementalConfig:
+    default_viewport: 'MD'
+    use_custom_title_classes: true'
+    css_framework: 'bulma'
+```
+
+## Add the extension
+To use the Grid extension on the page you want in your project, copy the following code and paste it to your extensions.yml
+
+```yaml
+Page\That\Uses\Grid:
+    extensions:
+        - DNADesign\Elemental\Extensions\ElementalPageExtension
+```
+
+### Switching between ElementalArea and default Content per page
+If you want to have the possibility to switch between a normal Content field or the ElementalArea, place the following extension on the page that also uses the grid:
+
+```yaml
+Page\That\Uses\Grid:
+    extensions:
+        - TheWebmen\ElementalGrid\Extensions\ElementalPageExtension
+```
+
+In your template, you can then use the following check to display the right content:
+
+```silverstripe
+<% if $UseElementalGrid && $ElementalArea %>
+    $ElementalArea
+<% else %>
+    $Content
+<% end_if %>
+```
+
+## Supported CSS frameworks
+Currently Bulma and Bootstrap (5) are the supported CSS frameworks that can be combined with this module.
+
+## Customizing the row, section and container CSS classes
+There are 3 extension point you can use to update which row, section and container CSS classes will be rendered in the templates.
```

Comment:
Just wondering; Is it also possible to "overwrite" the row/section/container `.ss` templates?

If this is possible; Maby we should seperate the "customizing" menu in a seperate docs and point out the ways you can customize it?

Reply:
For now i added a extra section in the readme about how to override.